lib/paymentrails/gateways/RecipientGateway.rb

```diff
@@ -25,10 +25,35 @@ def update(recipient_id, body)
     end
 
     def delete(recipient_id)
```

Comment:
Same as above, please add comment on the method clarifying arguments

Reply:
added in https://github.com/PaymentRails/ruby-sdk/pull/52/commits/7e87f13388e14242686a9b87d65c5995d35a034f